redisdb/manifest.json

```diff
@@ -11,5 +11,6 @@
     "windows"
   ],
   "version": "0.1.0",
-  "guid": "0e2f3ed1-d36b-47a4-b69c-fedb50adf240"
+  "guid": "0e2f3ed1-d36b-47a4-b69c-fedb50adf240",
+  "max_agent_version": "6.0.0"
```

Comment:
Hmm maybe. We might want to eliminate it but I think it's a good idea to have all of them stop at 6 for now